tutorials/R/R-skills/using-r-markdown/Rmd02-RMarkdown/Rmd02-RMarkdown.Rmd

```diff
@@ -245,7 +250,7 @@ The initial line of a code chunk must appear as:
  ```</code></pre>
 
 The `r` part of the chunk header identifies this chunk as an R code chunk and is 
-mandatory. Next to the `{r}`, there is a chunk name. This name is not required 
+mandatory. Next to the `{r`, there is a chunk name. This name is not required 
```

Comment:
Just fyi if interested: giving a code chunk name also defines the names for any of the rfigs produced in that chunk